src/interface_r/R/install.R

```diff
@@ -0,0 +1,16 @@
+#' Install h2o4gpu from a path to wheel file
+#' @export
+install_h2o4gpu <- function(wheel_path) {
+  # TODO: Support other install methods
```

Comment:
Other install methods would be virtualenv, which is what most devs for this project use.

Reply:
This will involve a little more work. I'll work on those in future PRs